ui/raidboss/data/06-ew/raid/p9n.txt

```diff
@@ -1,6 +1,259 @@
 ### P9N: Anabaseios: The Ninth Circle
 
+# Ability IDs that appear in this instance:
```

Comment:
This is nice! I think my only request would be to put this at the end of the timeline rather than the beginning, just so it's easier to scroll to the "real" stuff.